src/service/db.py

```diff
@@ -43,14 +61,20 @@ def _to_song_key(self, songName: str):
         )
 
     @with_cache("SongServiceByDB::get_format_tags")
-    def get_format_tags(self, spreadsheetId: str = "") -> Mapping[str, str]:
-        if spreadsheetId == "":
-            spreadsheetId = self.defaultSpreadsheetId
+    def get_format_tags(self, spreadsheet: str = "") -> Mapping[str, str]:
+        spreadsheetId = self.spreadsheetIds.get(spreadsheet, self.defaultSpreadsheetId)
 
         return self.service.songTemplateDB.get_format_tags(spreadsheetId)
 
-    def save_song(self, song: Song, spreadsheetId: str = ""):
-        if spreadsheetId == "":
-            spreadsheetId = self.defaultSpreadsheetId
+    @with_cache("SongServiceByDB::get_all_format_tags")
+    def get_all_format_tags(self) -> Mapping[str, str]:
+        return {
+            actor: style
+            for group, _ in self.spreadsheetIds.items()
```

Comment:
Nitpick: `for group in self.spreadsheetIds.keys()`

Reply:
fixed